admin_guide/metrics_guidlines.adoc

```diff
@@ -0,0 +1,83 @@
+[[metrics-guidelines]]
+= OpenShift Container Platform (OCP) Metrics Guidlines
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+OpenShift Container Platform Metrics enable OCP users and administrators to monitor
+pods in OCP installation and quickly get information about specific pod resource utilization
+and based on that decide is it necessary to give additional resources to specific application pods in order to
+assist with capacity planning and performance analysis efforts.
+
+== Metrics Guidelines
+This document gives guidlines for OCP metrics deployment. For specific OCP metrics configuration
+aspets refer to OCP documentation ( Link to OCP v3.4 Administation guide )
+
+=== OpenShift Container Platform Metrics Node Capacity Planning
+
+Running OCP metrics nodes is recommended on separate OCP nodes than application ones.
+For best performance, either ensure OCP metrics pods run on `infra` nodes, or OCP metrics pods ran on
+dedicated OCP nodes
+In our tests, metrics pods were collocated with other  `infra` pods and OCP node capacity was 8 CPUs / 32 GB of memory
+This recommendation is based on experiences gathered during testing of OpenShift Container Platform metrics installation
+
+=== Storage recommendations for OpenShift Metrics Deployment
+
+It is necessary prior to metrics deployment to plan storage space for Cassandra
+pod within OpenShift Container Platform metrics. All metrics data are written
```

Comment:
All metric data is written to the Cassandra database and it will grow over time.  The growth rate and size of the database is...